crates/js-sys/src/lib.rs

```diff
@@ -483,6 +484,30 @@ extern "C" {
     pub fn value_of(this: &Boolean) -> bool;
 }
 
+impl From<bool> for Boolean {
+    fn from(b: bool) -> Boolean {
+        Boolean::unchecked_from_js(JsValue::from(b))
+    }
+}
+
+impl From<Boolean> for bool {
+    fn from(b: Boolean) -> bool {
+        b.value_of()
```

Comment:
This can probably be implemented in a more optimized way, but we can save that for later.